featurebyte/service/base_document.py

```diff
@@ -766,6 +768,17 @@ async def _check_document_unique_constraints(
                 resolution_signature=resolution_signature,
             )
 
+    @staticmethod
+    def _check_document_modifiable(document: Document) -> None:
+        if document.block_modification_by:
+            block_modification_by = [
+                f"{item.asset_name}(id: {item.document_id})"
+                for item in document.block_modification_by
+            ]
+            raise DocumentModificationBlockedError(
+                f"Document {document.id} is blocked from modification by {block_modification_by}"
```

Comment:
might be useful to include potential steps on how to get unblocked if that's something users might want to do after getting this error.

Reply:
ah gotcha.